billinge.mplstyle

```diff
@@ -0,0 +1,40 @@
+#Author: Christopher J. Wright
+
+lines.linewidth : 2.0
+
+patch.linewidth: 0.5
+patch.facecolor: blue
+patch.edgecolor: eeeeee
+patch.antialiased: True
+
+text.hinting_factor : 8
+
+mathtext.fontset : cm
+
+axes.facecolor: eeeeee
+axes.edgecolor: bcbcbc
+#axes.grid : True
+axes.titlesize: x-large
+axes.labelsize: large
+axes.prop_cycle: cycler('color', ['348ABD', 'A60628', '7A68A6', '467821', 'D55E00', 'CC79A7', '56B4E9', '009E73', 'F0E442', '0072B2'])
+
+grid.color: b2b2b2
+grid.linestyle: --
+grid.linewidth: 0.5
+
+legend.fancybox: True
+legend.framealpha: .3
+
+xtick.direction: in
+ytick.direction: in
+
+image.cmap: viridis
+image.interpolation : none
+
+font.family: normal
```

Comment:
@CJ-Wright  what is `font.family` normal? Does this default to Bitstream Vera Sans? We need to specify a font that everyone has. We can easily have every non Windows user in the group install the microsoft core fonts library, so that all the usual serif fonts are on there. To check what fonts you have, use this:

```
import matplotlib as mpl
font_paths = mpl.font_manager.findSystemFonts()
font_objects = mpl.font_manager.createFontList(font_paths)
font_names = [f.name for f in font_objects]
print font_names
```

Then the font can be set inside this mplstyle sheet you're making. Or, in any jupyter notebook with `plt.rcParams['font.family']='FontXXX'`

I dislike this billinge.mplstyle as it stands. But, these figure style issues are quickly becoming frustrating/distracting/important, so I'd like to make a request. Could you generate a small variety of figures using different style templates (including seaborn if possible)? Then, I think the best way forward, is to **not** make this a democratic decision, and instead ask @sbillinge to pick the one he feels does the job best, and then we stick to that. If I can help with this, or if you think generating a variety of example figs is not the best strategy, let me know.